cvat-ui/src/components/actions-menu/dump-submenu.tsx

```diff
@@ -19,15 +19,17 @@ interface Props {
     menuKey: string;
     dumpers: any[];
     dumpActivities: string[] | null;
+    taskDimension: string;
```

Comment:
Enum

Reply:
Corresponding above, something like this:
```
taskDimension: TaskDimension.D2 | TaskDimension.D3
```

Reply:
Changed code accordingly.

Reply:
Lets' use unified name here **DimensionType** (like in Python code)

Reply:
@bsekachev can we use **cvat-core/src/enums.js** here? (it might be a hard link of 2 separate modules, I believe)